### PR TITLE
Adminwho now caches feedback links until admins are reloaded (and also works again)

### LINKED ISSUE #67150: Adminwho is slower than it needs to be after feedback links
After https://github.com/tgstation/tgstation/pull/66909, every admin feedback link is requested serially.

This means that, if there are 5 admins, then the process currently goes:

1. Get admin 1's feedback link. Yield.
2. Get admin 2's feedback link. Yield.
3. Get admin 3's feedback link. Yield.
4. Get admin 4's feedback link. Yield.
5. Get admin 5's feedback link. Yield.

This is slower than it has to be. There are 2 obvious ways to fix this while staying in the boundaries of the "TTL for data from the database is 10 seconds" rule.

One is simply spinning up a bunch of async calls for the database.

The second, more correct way, is combining them into one SELECT call, such as `SELECT ckey, feedback FROM admin WHERE ckey = 'admin1' OR ckey = 'admin2'

### LINKED ISSUE #67360: Untrustworthy code ocelot broke feedback links in adminwho.
## Round ID:
N/A

## Testmerges:
N/A

## Reproduction:
#67261 broke admin feedback links.

image

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Fixes #67150.
Fixes #67360.

https://user-images.githubusercontent.com/35135081/170387509-4fd5bcf7-eb63-46f2-9976-7f852788b9e7.mp4

## Why It's Good For The Game

Adminwho, especially with a lot of admins on, was taking 5-10 seconds to complete. This is unacceptable.

## Changelog

:cl:
fix: Fixed Adminwho taking several seconds to resolve.
/:cl:

### CODE REVIEW

**Review by LemonInTheDark — APPROVED** [review]

File: code/modules/admin/holder2.dm [hunk 1]
```diff
@@ -172,7 +172,9 @@ GLOBAL_PROTECT(href_token)
 
 /// Returns the feedback forum thread for the admin holder's owner, as according to DB.
 /datum/admins/proc/feedback_link()
-	if(world.time - last_forum_access_time <= 10 SECONDS)
+	// This intentionally does not follow the 10-second maximum TTL rule,
+	// as this can be reloaded through the Reload-Admins verb.
+	if(!isnull(cached_forum_link))
 		return cached_forum_link
 
 	last_forum_access_time = world.time
```
Comment (Mothblocks) [inline code/modules/admin/holder2.dm:180]: Need to remove this

**Comment by Mothblocks** [comment]
I'm not happy with how this is implemented actually

**Comment by Mothblocks** [comment]
Also it doesn't appear to work so there's that

**Comment by Timberpoes** [comment]
As per ivory tower discussions, I'll crosslink this with https://github.com/tgstation/tgstation/issues/67360 which I believe you intended to fix in this PR as well.

**Pull request merged**